lessons/2 - git time/homework.html

```diff
@@ -0,0 +1,96 @@
+<html>
+
+<head>
+	<title>Javascript orientation session!</title>
+	<link rel="stylesheet" href="../../reveal.js/css/reveal.css">
+	<link rel="stylesheet" href="../../reveal.js/css/theme/moon.css">
+	<link rel="stylesheet" href="../../reveal.js/lib/css/zenburn.css">
+	<link rel="JavaScript" href="../../reveal.js/plugin/markdown/marked.js">
+	<link rel="JavaScript" href="../../reveal.js/plugin/markdown/markdown.js">
+	<style>
+		.reveal pre {
+			margin: 0;
+		}
+
+		.monospace {
+			font-family: monospace !important;
+		}
+
+		.illustration {
+			border: 0px
+		}
+
+		.reveal li {
+			margin-bottom: 0.5em;
+		}
+
+		.reveal pre {
+			width: 100%;
+		}
+
+		.reveal pre code {
+			cursor: pointer;
+		}
+
+		.reveal li pre {
+			width: fit-content;
+		}
+
+		.logo {
+			position: fixed;
+			/* Sit on top of the page content */
+			z-index: 2;
+			/* Specify a stack order in case you're using a different order for other elements */
+		}
+	</style>
+</head>
+
+<body>
+	<a href="https://www.redi-school.org/"><img src="../../redi-school-logo.jpg" class="logo" /></a>
+	<div class="reveal">
+		<div class="slides">
+			<section>
+				<h1>Homework</h1>
+      </section>
+			<section>
+				<h3>Homework: Start setting up your project</h3>
+				<ul class="fragment">
+					<li>Create a NEW branch and add to your project</li>
+					<ul>
+						<li>css folder with main.css file</li>
+						<li>js folder with main.js file</li>
+					</ul>
+					<li>stage changes, commit, push to branch</li>
+					<li>Create new pull request</li>
+					<ul>
+						<li>select ALL teachers as reviewers</li>
+						<li>assign pull request to me 'aberezhk'</li>
+					</ul>
+					<li>DO NOT MERGE - teacher will do it</li>
```

Comment:
@aberezhk Why teachers would merge it?) I think if it was approved by at least 1 teacher (maybe 2) then student can merge it into master.

Reply:
Added slide with links to various git related materials (last slide in lecture and homework deck)